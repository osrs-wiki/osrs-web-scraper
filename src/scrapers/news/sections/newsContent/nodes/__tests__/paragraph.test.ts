import { MediaWikiBuilder } from "@osrs-wiki/mediawiki-builder";
import parse from "node-html-parser";

import paragraphParser from "../paragraph";

describe("p node", () => {
  test("Paragraph should parse and render", () => {
    const root = parse("<p><b>test</b></p>");
    const builder = new MediaWikiBuilder();
    builder.addContents([paragraphParser(root.firstChild)].flat());
    expect(builder.build()).toMatchSnapshot();
  });
});
