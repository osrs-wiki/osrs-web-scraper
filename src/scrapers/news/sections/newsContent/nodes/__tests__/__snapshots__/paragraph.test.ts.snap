// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`p node Paragraph should parse and render 1`] = `
"'''test'''
"
`;
